src/api/ratelimits.mdx

```diff
@@ -26,9 +26,9 @@ Every API request response includes the following headers:
 - `X-Sentry-Rate-Limit-Reset`
   - The time when the next rate limit window begins and the count resets, measured in UTC seconds from epoch
 - `X-Sentry-Rate-Limit-ConcurrentLimit`
-  - The maximum number of concurrent requests allowed within the window
+  - The maximum number of concurrent ("in flight") requests allowed at any given time
```

Comment:
🤔 

'concurrent' does not mean 'in flight', just means happening at the same time.
So this _could_ just be `The maximum number of concurrent, in flight requests allowed at any given time`. But this also becomes slightly redundant since `in flight` and any given time are both sort of timing-specific participles 🤔 

'concurrent' and 'at the same time' are redundant too.
in-flight is a given and is unnecessary as any request in the pipeline is "in flight" already.

Reply:
if the confusion is around the term `concurrent` we should just remove that from the doc and clarify it in more laymans terms

Reply:
The confusion around concurrent rate limiting typically stems from the fact that it is not actually window based. 

In flight is redundant for sure (the specific report was resolved by explaining these are all requests currently in-progress/in-flight).

I am open for using whatever describes concurrency in more laymans terms.

The root problem is the original language "within the window"